ci/gpu/build.sh

```diff
@@ -86,12 +90,12 @@ set +x
 gpuci_logger "Check GPU usage"
 nvidia-smi
 
-gpuci_logger "GoogleTest for raft"
+gpuci_logger "GoogleTest for libraft"
 GTEST_OUTPUT="xml:${WORKSPACE}/test-results/raft_cpp/" $CONDA_PREFIX/bin/libraft/gtests/test_raft
 
-gpuci_logger "Python pytest for pyraft"
-cd "$WORKSPACE/python/raft/raft/test"
-pytest --cache-clear --junitxml="$WORKSPACE/junit-pyraft.xml" -v -s
+gpuci_logger "Python pytest for raft-dask"
```

Comment:
Question a bit unrelated to the PR, wouldn't it be better to run the `pylibraft` pytests first, so that tests run from lower to higher level?

Reply:
Ah! That's a good point. Assuming CI Passes, let me merge this guy over and I'll create a follow-on to fix that.